src/deviceApiCalls/schemas/sendJSPixel.params.json

```diff
@@ -37,6 +37,29 @@
       ],
       "additionalProperties": false
     },
+    {
+      "properties": {
+        "pixelName": {
+          "type": "string",
+          "const": "autofill_credentials_import_prompt_shown"
+        }
+      },
+      "required": [
+        "pixelName"
+      ],
+      "additionalProperties": false
+    },    {
+      "properties": {
+        "pixelName": {
+          "type": "string",
+          "const": "autofill_import_credentials_flow_started"
```

Comment:
As mentioned in the code above, we should fire this from the native side, therefore we can remove it from here.

Reply:
I think that's a good idea! Also because it seems like firing from JS side appends a bunch of legacy pre-fixes. We could get around that with native taking care of this.

Reply:
Removed https://github.com/duckduckgo/duckduckgo-autofill/pull/631/commits/03366ad416641e15bf246cddafef8d9b5cbee473